src/Resources/app/storefront/src/scss/base.scss

```diff
@@ -74,3 +75,33 @@
 .adyen-payment-action-container {
     height: 100%;
 }
+
+select{
```

Comment:
This CSS selector is too broad and it affects all of the loaded `select` HTML elements. That needs to use id or class.